Gruntfile.js

```diff
@@ -197,7 +197,7 @@ module.exports = function (grunt) {
     typescript: {
       base: {
         src: ['<%= yeoman.app %>/scripts/{,*/}*.ts'],
-        dest: '.tmp/scripts',
+        dest: '.tmp/compiled.js',
```

Comment:
Is there a better method of serving modules dynamically? 

I don't really like this concat everything into one file (if I understand what this does).